### ISSUE TITLE
Bugwarrior recreates task if task is completed

### ISSUE THREAD

**Issue by StephanMeijer** [ralphbean/bugwarrior#769]
On reassignment, JIRA creates a completely new task instead of reusing the old one.

**Comment by StephanMeijer** [comment]
Other tasks do this as well: As soon as a task/issue is completed and then being put in progress again, it won't be set on pending anymore. A new (duplicate) task will be added instead. This is a bugwarrior-wide issue.

**Comment by StephanMeijer** [comment]
I suspect `find_taskwarrior_uuid` can only find tasks with `status: pending`.

**Comment by StephanMeijer** [comment]
This suspicion might be correct:

```python
            results = tw.filter_tasks({
                'and': [("%s.is" % key, issue[key]) for key in key_list],
                'or': [
                    ('status', 'pending'),
                    ('status', 'waiting'),
                ],
            })
```

**Comment by ryneeverett** [comment]
Resolved via #779.

**Comment by StephanMeijer** [comment]
Thanks for picking this up @ryneeverett
